ex04.ex

```diff
@@ -39,7 +39,12 @@ defmodule Ex04 do
       [ 1, 2, 3, 4, 5 ]
 
   """
-  def reverse . . . "your code"
+  def reverse([]) do
```

Comment:
# 3

You  can remove the first definition altogether if you replace the [h|t] which just list in the second:

~~~ elixir
def reverse(list), do: reduce list, [], &([&1|&2])
~~~